### PR TITLE
Fixes #7988 - unify foreman postinst files

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: debian/focal/foreman/foreman.install [hunk 1]
```diff
@@ -7,6 +7,7 @@ script/foreman-rake                      usr/sbin
 script/foreman-tail                      usr/sbin
 debian/foreman-plugin.mk                 usr/share/foreman
 debian/foreman-plugin.postinst-lib       usr/share/foreman
+debian/foreman.postinst-lib              usr/share/foreman
```
Comment (ekohl) [inline debian/focal/foreman/foreman.install:10]: I don't see this change for bullseye. Am I missing something?

**Review by ekohl — COMMENTED** [review]
This looks like a good improvement.

File: debian/focal/foreman/foreman.install (hunk 1 above)
Comment (evgeni) [reply debian/focal/foreman/foreman.install:10]: You are not, I am ;)

**Comment by evgeni** [comment]
[test deb]

**Comment by evgeni** [comment]
[test deb]

**Review by ehelms — APPROVED** [review]

**Pull request merged**